docs/05.PORT-API.md

```diff
@@ -83,13 +84,24 @@ typedef struct
  *
  * @return true  - if success
  *         false - otherwise
+ *
+ * Note:
+ *      This port function is called by jerry-core when
+ *      CONFIG_DISABLE_DATE_BUILTIN is _not_ defined. Otherwise this function is
+ *      not used.
```

Comment:
Note should be before return (similar cases below).

Reply:
If that's the case then the doc of `jerry_port_fatal` will have to be fixed as it has `@param` before the note. Should we take that approach?